### DISCUSSION TITLE
Faire un bouton pour passer d'une langue à une autre (exemple Francais<>Anglais)

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#607] [Q&A]
Bonjour,

Est-ce que quelqu'un a déjà fait un bouton pour pouvoir changer la langue dans le synapps et passer par exemple de Français à anglais ?
Il faudrait passer probablement par un petit script en OnPropertyChanged mais quel est la ligne de code à saisir pour forcer le changement de la langue de prévisualisation ?

Merci d'avance

**Comment by Raphwit** [comment]
Hello, si je comprend bien tu voudrais changer la langue de la synapp directement depuis la synapp ?

**Reply by JeremieDENAEGEL** [reply]
Oui, je voudrais faire un bouton pour changer la langue dans le synapps.

Chez certains clients (Usine, sites transfrontaliers, Siège de groupe international, ...), il est parfois nécéssaire et demandé de pouvoir changer la langue selon l'utilisateur.

Par exemple, une majorité des sites internet en belgique ont un bouton pour passer du français au néerlandais ou à l'anglais. De mémoire en catalogne, cela est la même chose entre l'espagnol et le catalan.
Pour notre part, nous avons déjà eu quelques sites dans le Nord où l'administrateur du site voulait ses synoptiques en anglais et pour les autres usagers, nous avions laissé en français.
